src/internal/dockertestenv/postgres.go

```diff
@@ -169,7 +169,7 @@ func EnsureDBEnv(ctx context.Context) error {
 		PortMap: map[uint16]uint16{
 			30228: 5432,
 		},
-		Image: "postgres:13.0-alpine",
+		Image: "postgres:15.5",
```

Comment:
Don't have strong opinions either way.  But this new tag is not alpine.  So this new image is probably considerably larger.  For development is won't matter too much, the images are cached. But it might be a consideration for CI.

Reply:
updated to 15-alpine 👍 . good call out